### ISSUE TITLE
`ver` is not defined in  org-readme-sync

### ISSUE THREAD

**Issue by CestDiego** [mattfidler/org-readme#14]
I get this error:
```
call-interactively: Symbol's value as variable is void: ver
```
and reading the source code i think this is where it comes from: https://github.com/mattfidler/org-readme/blob/master/org-readme.el#L1754

**Comment by mattfidler** [comment]
Thanks for the catch.  I'm unsure if this fix works, but I think it should.  I actually don't use org-readme very much anymore.

**Comment by CestDiego** [comment]
;_; @mattfidler why is that? did you find other way of doing the same?

**Comment by mattfidler** [comment]
No, I haven't.  However, mostly have spent time on ergoemacs-mode.  Since there is a whole web-page devoted to this, I would have to think about how to integrate o-blog to create the web-page/readme.  This is a bit low on my priority list, but I think it would be awesome to be able to do that.
